tableauserverclient/server/endpoint/databases_endpoint.py

```diff
@@ -93,15 +93,15 @@ def update_permission(self, item, rules):
 
     @api(version='3.5')
     def delete_permission(self, item, rules):
-        return self._permissions.delete(item, rules)
+        self._permissions.delete(item, rules)
```

Comment:
I reviewed the underlying methods, and I know what I was going for, but thanks for pointing out it didn't work :P

DELETE should get a 204 back from the Server, but we don't return it (https://github.com/tableau/server-client-python/blame/master/tableauserverclient/server/endpoint/endpoint.py#L89). 

I think we should though the plumbing might not be worth it, I'll take a look in another item

Reply:
It was working, I just noticed that the underlying methods weren't returning anything so I removed it here. I think returning the 204 would be nice too, since currently it's a silent operation unless theres is an error.